.github/workflows/main.yml

```diff
@@ -2,8 +2,67 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  test:
-    name: Test
+  buildlibc:
+    name: Build libc
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: Install clang (Windows)
+      shell: bash
+      run: |
+          curl -fsSLO https://releases.llvm.org/8.0.0/LLVM-8.0.0-win64.exe
```

Comment:
So this .exe is secretly a 7zip formatting file?  Strange.

Reply:
I have no idea why this works. I copy-pasted it from somewhere on the internet, and appears to do just fine. I don't have a windows machine on which to even try it, so I only know that it works here :)